### DISCUSSION TITLE
how can i use params with slash?

### DISCUSSION THREAD

**Discussion by sasha172017** [adonisjs/core#1817] [Help] [answered]
i need get param with slash, for example http://0.0.0.0/download/user/avatar/1603899939785.jpg, i need to receive string "avatar/1603899939785.jpg" how params, is it possible?

**Comment by thetutlage** [comment, marked as answer]
Here's how you can get it

```ts
Route.get('/uploads/*', async ({ params }) => {
  return Application.tmpPath(...params['*'])
})

```

**Reply by sasha172017** [reply]
thank you, can you add this in documentation?
